cypress/support/gutenberg/plugins/blockSnapshots.js

```diff
@@ -166,6 +166,39 @@ class BlockSnapshots {
 			} )
 		} )
 	}
+
+	/**
+	 * Enqueue all stubbed html contents to the frontend.
+	 * Individually assert its content.
+	 */
+	 assertFrontendContent() {
```

Comment:
do we need to add this?

Reply:
Sorry! I forgot to remove this. I'll remove it now